monkey/infection_monkey/consts.py

```diff
@@ -0,0 +1 @@
+USERNAME_PREFIX = "somenewuser"

```

Comment:
I don't think it's worth creating a new file just for this. I suppose mimikatz credential collector should just import from PBA, since that's the source of users we don't want to add.

Reply:
Don't couple the mimikatz collector to the PBAs!